src/main/java/com/backbase/ct/bbfuel/setup/CapabilitiesDataSetup.java

```diff
@@ -226,10 +228,10 @@ private void ingestPockets() {
                 .filter(user -> user.getExternalId().equalsIgnoreCase("user"))
                 .filter(user -> user.getRole().equalsIgnoreCase("retail"))
                 .filter(user -> StringUtils.isNotEmpty(user.getProductGroupNames()
-                        .stream()
-                        .filter(productGroupName -> productGroupName.equals("Retail Pocket"))
-                        .findFirst()
-                        .get()))
+                    .stream()
+                    .filter(productGroupName -> productGroupName.equals("Retail Pocket"))
```

Comment:
It would be nice if you declare "Retail Pocket" as constant.

Reply:
done